code_snippets/api-billing-usage-timeseries.sh

```diff
@@ -0,0 +1,12 @@
+api_key=9775a026f1ca7d1c6c5af9d94d9595a4
+app_key=87ce4a24b5553d2e482ea8a8500e71b8ad4554ff
+
+start_hr=$(date -v -10d +%Y-%m-%dT%H)
+end_hr=$(date -v -10d +%Y-%m-%dT%H)
```

Comment:
@jimabramson Are these intentionally the same? Shouldn't start time be before end time?